.travis.yml

```diff
@@ -4,8 +4,8 @@ services:
   - docker
 
 env: 
-  #- IMAGE_NAME=stacks_dlrs_mkl DIR=dlrs/clearlinux/tensorflow/mkl/
-  - IMAGE_NAME=stacks_dlrs_oss DIR=dlrs/clearlinux/tensorflow/oss/
+  - IMAGE_NAME=stacks_dlrs_oss DIR=dlrs/clearlinux/tensorflow/oss/ BUILD_ARGS='--build-arg clear_ver="30650"'
```

Comment:
Let's add this change in a different PR if needed, we should be updating these dockerfiles soon, so BUILD_ARGS might not be necessary at all in the future.
+1 on the mers changes

Reply:
+1